tests/scripts/deploy_admission_controller.sh

```diff
@@ -11,16 +11,45 @@ BASE_DIR=$(cd "$(dirname "$0")"; pwd)
 WEBHOOK_CONFIG_NAME="rook-ceph-webhook"
 SERVICE_NAME="rook-ceph-admission-controller"
 
-INSTALL_SELF_SIGNED_CERT=true
 echo "$BASE_DIR"
 
-if [ "${INSTALL_SELF_SIGNED_CERT}" == true ]; then
-	"${BASE_DIR}"/webhook-create-signed-cert.sh --namespace ${NAMESPACE}
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.1.0/cert-manager.yaml
```

Comment:
Let's make `v1.1.0` a variable so we can easily bump it.

Reply:
Done.